src/main/java/frc/robot/Constants.java

```diff
@@ -18,13 +18,19 @@ public final class Constants {
   public static final int FLIGHTSTICKUSBINDEX = 1;
   /// CAN Bus Devices
   // Drive Train Motors
-  public static final int MOTORFRONTRIGHTID = 1;
-  public static final int MOTORBACKRIGHTID = 2;
-  public static final int MOTORFRONTLEFTID = 3;
-  public static final int MOTORBACKLEFTID = 4;
+  public static final int MOTORFRONTLEFTID = 11;
+  public static final int MOTORBACKLEFTID = 12;
+  public static final int MOTORFRONTRIGHTID = 13;
+  public static final int MOTORBACKRIGHTID = 14;
+  // Arm movement motors
+  public static final int ARMXMOTORID = 21;
+  public static final int ARMYMOTORID = 22;
```

Comment:
I thought we only had one motor for the arm?

Reply:
I thought kian told me something else, i'll reach out to him.

Reply:
Thanks!